app/views/smart_answers/custom_result.html.erb

```diff
@@ -8,7 +8,17 @@
 <% end %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div
+    class="govuk-grid-column-two-thirds"
+    data-module="ga4-auto-tracker"
+    data-ga4-auto='{
+      "event_name": "smart_answer_complete",
```

Comment:
This might possibly need to be `form_complete` - I've checked the implementation guide and the code snippet says `event_name: smart_answer_complete` but the table below giving examples of the data values has `event_name` as `form_complete`. I've sent a quick message to the PA's to check.

Reply:
Just heard back from the PA's and they've asked for this to be `form_complete`

Reply:
Great spot @JamesCGDS, thanks for checking this out. Have updated the three instances of this now.